packages/main/src/Card.js

```diff
@@ -3,6 +3,8 @@ import URI from "@ui5/webcomponents-base/src/types/URI";
 import Bootstrap from "@ui5/webcomponents-base/src/Bootstrap";
 import ShadowDOM from "@ui5/webcomponents-base/src/compatibility/ShadowDOM";
 import { isIconURI } from "@ui5/webcomponents-base/src/IconPool";
+import { isSpace, isEnter, } from "@ui5/webcomponents-base/src/events/PseudoEvents";
+import Function from "@ui5/webcomponents-base/src/types/Function";
```

Comment:
there is a native type function so I think we can skip this?

Reply:
There is, but the metadata validation would not pass for now. As this refactoring is started by Vladi, I will change it to native Function once it's done.